### PR TITLE
Jay oceans patch 1

### PR DESCRIPTION
**Pull request by JayOceans**
sorry, i'm rusty at github

### CODE REVIEW

File: _pages/equipment.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: Equipment
 ---
 
+Intentionallyyyyyyy mispelleddddd
```
Comment (mflaxman) [inline _pages/equipment.md:5]: I assume this is a mistake?

File: _pages/known-issues/verify-receive-address.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ Smart hardware wallets implement a defense against this where they register all
 The bullet-proof solution is to verify your receive addresses on a quorum of trusted displays.
 Not only does this eliminate the risk, but it serves as a redundant check when making a deposit.
 While perhaps a little counter-intuitive, this is an excellent safety feature.
-Smart single-key signature users also must redundantly verify their receive addresses, but doing so in a single-key setup adds new issues due to the inherent single point of failure in single-key signature schemes;
-you now have your seed floating around in multiple places for verification, and suffering a breach in any one of these is enough to lose all your funds!
+Smart single-key signature users also must redundantly verify their receive addresses, but doing so in a single-key setup adds a new privacy issue do to sharing xpubs in a watch-only wallet.
```
Comment (mflaxman) [inline _pages/known-issues/verify-receive-address.md:28]: I don't understand what this means, can you clarify?

Comment (JayOceans) [reply _pages/known-issues/verify-receive-address.md:28]: sorry, let me try...
to verify addresses, all one needs to do is create a watch-only wallet & compare a few addresses...
there's no need to restore a new wallet from seed in order to verify addresses!...
so the issue, is not about risk of losing coins... it's only about leaking xpubs... (and actually, doing this all Offline, in electrum for example, is easy & has no leaks...

Comment (mflaxman) [reply _pages/known-issues/verify-receive-address.md:28]: You need to verify seed -> xpub on multiple devices (which exposes your seed on multiple devices) or you have a single-point of failure.

**Comment by mflaxman** [comment]
This has turned into an issue discussion (issues welcome), closing this for now as PRs are for pull requests and this is not in a mergable state.

Thanks!

**Pull request closed without merging**